src/components/Section/Identification/ApplicantName/ApplicantName.jsx

```diff
@@ -2,18 +2,66 @@ import React from 'react'
 import { i18n } from '../../../../config'
 import schema from '../../../../schema'
 import validate from '../../../../validators'
-import SubsectionElement from '../../SubsectionElement'
 import { Name, Field } from '../../../Form'
 
-export default class ApplicantName extends SubsectionElement {
+import { reportCompletion } from '../../../../actions/ApplicationActions'
+
+import connectIdentificationSection from '../IdentificationConnector'
+
+const sectionConfig = {
```

Comment:
This could be hoisted to a top level config

Reply:
i.e. https://github.com/18F/e-QIP-prototype/pull/1439/files#diff-8ce44bb418dc407a198758d1fe38422e